src/pages/paths.ts

```diff
@@ -0,0 +1,38 @@
+// This is just an array of the app's defined paths that can be used in our Cloudflare Functions.
+// Do not add any imports to this file.
+// The array is kept up to date via the tests in src/pages/paths.test.ts
+
+export const paths = [
```

Comment:
do we need maintain this as a hardcoded array or can we just map paths from `routes` the same way you do in the test above?

Reply:
yeah, we need a copy that doesn't pull in a bunch of other dependencies because it doesn't compile in the functions code